content/english/blog/new_logo_2025.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "PyPodcats have a new logo"
+meta_title: "PyPodcats have a new logo"
+description: "Finally after 1 year, we finally have a logo"
+date: 2025-06-17T05:00:00Z
+categories: ["Blog Post",]
+author: "Georgi"
+tags: ["python", "community", "women in tech", "women in Python", "women in open source","python underrespresnted groups"]
+draft: false
+image: "/images/logo-blog.png"
+---
+
+## PyPodcats Have a New Logo
+After a year of placeholder art and indecision, we finally have a real logo. And yes—it has a cat. On a mic. With a snake. We promise it makes sense.
+Our original stand-in was a charming sticker from Mariatta’s [“#TypoOfTheDay”](https://mariatta.ca/posts/typo_of_the_day/) series. It wasn’t meant to be a logo, but we were so eager to get *Hidden Figures of Python* out into the world that we just rolled with it. MVP: Minimum Viable Purr.
+Many drafts, discussions, and digital furballs later, our team chose the new design: a laid-back cat draped over a retro microphone, with a happy snake coiled in—our nod to the Python community that brought us all together.
+We printed our first batch of stickers just in time for PyCon US this past May. They were glittery. They were gorgeous. They were gone in a flash. Sorry if you missed them—we had a tight budget and only printed a small run using our own pocket money. Blame the import tax on sparkles.
```

Comment:
Insert the new logo as an image before this paragraph, and say "Check our new logo. What do you think?"

Reply:
do you mean the sticker? Because the logo is already on the blogpost top?

Reply:
Ah yes, if there is picture of the glittery stickers, it would be great. The blog post is looking great! Thank you!